src/web/components/Card/components/StandfirstWrapper.tsx

```diff
@@ -4,20 +4,17 @@ import { css } from 'emotion';
 import { body } from '@guardian/src-foundations/typography';
 
 type Props = {
-    children: JSXElements;
+    children: string;
 };
 
 export const StandfirstWrapper = ({ children }: Props) => (
     <div
         className={css`
             display: flex;
             flex-direction: column;
-            justify-content: space-between;
 
-            ${body.small({
-                fontWeight: 'light',
-            })};
-            line-height: 20px;
+            ${body.small()};
+            font-size: 14px;
```

Comment:
I'd stick with the design system, rather than override here I think? @SiAdcock ?

Reply:
I discussed this with Simon. The design system gives you 15px, not 14px and whilst 1px isn't much, changing it would cause wrapping diffs, which is bad

The agreement was to see how often we need to override to 14 and then decide if we need to change the design system to fit the site.

Reply:
It's so interesting that we've chosen 15px as one of our body font sizes, given how [frontend generally uses 14px](https://github.com/guardian/frontend/blob/master/static/src/stylesheets/pasteup/typography/_src.scss). 

`body.small` is used in [only a few places](https://github.com/guardian/dotcom-rendering/search?q=%22body.small%22&unscoped_q=%22body.small%22). `textSans.small`, which is also 15px, is [used more frequently](https://github.com/guardian/dotcom-rendering/search?q=textSans.small&unscoped_q=textSans.small) but I still believe the impact will be small if we decided to change the size from 15px to 14px. Perhaps we can discuss next week?